src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -16,6 +16,7 @@
   public static final String BOTTOM_LB = "BOTTOM_LEADERBOARD";
   public static final String MOBILE_BOTTOM_LB = "bottom_leaderboard";
   public static final String TOP_LB = "TOP_LEADERBOARD";
+  public static final String HIVI_TOP_LB = "top_leaderboard"; // revert to hivi_leaderboard after ae3 rollout to XX
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)